tools/tools.go

```diff
@@ -0,0 +1,13 @@
+//go:build tools
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/onsi/ginkgo/v2/ginkgo/generators"
+	_ "github.com/onsi/ginkgo/v2/ginkgo/internal"
```

Comment:
Why not just import "github.com/onsi/ginkgo/v2/ginkgo"? Importing an "internal" package looks very wrong.

Reply:
I guess this was some quirk with `go mod tidy` missing some dependencies, so we couldn't run ginkgo with `go run github.com/onsi/ginkgo...`